src/lifinity/infos.ts

```diff
@@ -13,9 +13,15 @@ const DIGIT = new BN(10000000000);
 let infos: IInstancePool;
 
 infos = class InstanceLifinity {
-  static async getAllPools(connection: Connection): Promise<types.PoolInfo[]> {
-    const lifinityAccounts = await getMultipleAccounts(connection, LIFINITY_ALL_AMM_ID);
-
+  static async getAllPools(connection: Connection, page?: PageConfig): Promise<types.PoolInfo[]> {
+    let lifinityAccounts = await getMultipleAccounts(connection, LIFINITY_ALL_AMM_ID);
+    lifinityAccounts.sort((a, b) => a.pubkey.toBase58().localeCompare(b.pubkey.toBase58()));
+    if (page) {
```

Comment:
1. Perhaps this piece of logic can be extracted to utils as a function? Ex: `Paginate`
2. Assign the batched accounts to a new variable (Ex: `pagedPoolAccounts`) instead of mutating `allV4AMMAccount`